tests/unit-tests/product/query.php

```diff
@@ -64,6 +64,12 @@ public function test_product_query_standard() {
 		$query->set( 'limit', 1 );
 		$results = $query->get_products();
 		$this->assertEquals( 1, count( $results ) );
+
```

Comment:
You should create a new product without a sku here:

```
$product3 = new WC_Product_Simple();
$product3->save()
```

Currently this unit test passes, but it's a false positive because the query matches on products that have '' as a sku.

Reply:
Thanks for the feedback and advice! I have made the changes as requested. It will now check the field exists and also that the field does not have a blank value